### PR TITLE
feat(qccollect_rna): Collect and evaluate rna data

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- Adds Qcinspect to the rna pipeline
- Evaluates
  - percentage_reads_with_adapter
  - percentage_reads_after_trimming
  - percentage_bp_after_trimming
  - percentage_uniquely_mapped_reads
  - fraction_duplicates
- Addresses #1371

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [x] Installation, unit and integration test suite pass

### Review:
- [x] Code review
- [x] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [x] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Cli/Mip/Analyse/Rd_rna.pm [hunk 1]
```diff
@@ -748,6 +748,57 @@ q{GATK VariantFiltration, window size (in bases) in which to evaluate clustered
...
+            documentation => q{File containing Qccollect evaluation metrics},
+            is            => q{rw},
+            isa           => Str,
+        )
+    );
+
+    option(
+        q{qccollect_regexp_file} => (
+            cmd_aliases => [qw{ qccref }],
+            cmd_tags    => [q{Default: qc_regexp_-v1.24-.yaml}],
```
Comment (henrikstranneheim) [inline lib/MIP/Cli/Mip/Analyse/Rd_rna.pm:774]:
```suggestion
            cmd_tags    => [q{Default: qc_regexp_-v1.25-.yaml}],
```

Comment (henrikstranneheim) [reply lib/MIP/Cli/Mip/Analyse/Rd_rna.pm:774]: Should the default be changed for the other pipelines as well?

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Rename frequency_annotation to variant_annotation
 - Removed option to run sambamba markduplicates for markduplicates recipe
 - Added SpliceAI annotation
+- Collect and evaluate QC metrics generated in the RNA pipeline
```
Comment (henrikstranneheim) [inline CHANGELOG.md:15]: Should we update the new picard version here as well?

Comment (jemten) [reply CHANGELOG.md:15]: Forgot about that. I'm working on implementing picard CollectRnaSeqMetrics and the version of picard that we have has a bug in that program. Anyway, the picard upgrade was meant for that PR. I'll add it to the changelog

File: lib/MIP/Cli/Mip/Analyse/Rd_rna.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Cli/Mip/Analyse/Rd_rna.pm:774]: Not critical, but for consistency yes

**Pull request merged**